src/xcode/ENA/ENA/Source/Services/PPAnalyticsCollector/Model/PPAnalyticsData.swift

```diff
@@ -55,15 +63,25 @@ extension SecureStore: PPAnalyticsData {
 		set { kvStore["submittedWithQR"] = newValue }
 	}
 
-	var currentRiskExposureMetadata: RiskExposureMetadata? {
+	var currentENFRiskExposureMetadata: RiskExposureMetadata? {
 		get { kvStore["currentRiskExposureMetadata"] as RiskExposureMetadata? ?? nil }
 		set { kvStore["currentRiskExposureMetadata"] = newValue }
 	}
 
-	var previousRiskExposureMetadata: RiskExposureMetadata? {
+	var previousENFRiskExposureMetadata: RiskExposureMetadata? {
 		get { kvStore["previousRiskExposureMetadata"] as RiskExposureMetadata? ?? nil }
 		set { kvStore["previousRiskExposureMetadata"] = newValue }
 	}
+	
+	var currentCheckinRiskExposureMetadata: RiskExposureMetadata? {
```

Comment:
Name of property different to name of kvStore key.

Reply:
Yeah because if I named it properly, we would need a migration.

Reply:
Isn't this a new field? 🤔

Reply:
ah now I got it. Renamed it to match the property